src/dotnet/APIView/APIViewWeb/Managers/APIRevisionsManager.cs

```diff
@@ -985,5 +987,18 @@ private async Task GenerateAPIRevisionInExternalResource(ReviewListItemModel rev
             }
             return result;
         }
+        private async Task<APIRevisionListItemModel> UpgradeAPIRevisionIfRequired(APIRevisionListItemModel revisionModel)
+        {
+            if (revisionModel == null)
```

Comment:
This is not my codebase so feel free to ignore me, but braceless if statements can be pretty bug prone as it's easy to misread when maintaining/updating.